starbook-backend/static/.dockerignore

```diff
@@ -0,0 +1,5 @@
+Dockerfile
```

Comment:
you dont need this file at all, and especially not serve it

Reply:
removed